addons/web/static/src/views/calendar/calendar_common/calendar_common_renderer.js

```diff
@@ -65,7 +67,7 @@ export class CalendarCommonRenderer extends Component {
     get options() {
         return {
             allDaySlot: this.props.model.hasAllDaySlot,
-            allDayText: _t("All day"),
+            allDayText: _t(""),
```

Comment:
@edi-odoo Hi, was this really intended? Translating empty strings doesn't seem to make sense, and it gives a warning when exporting the source terms. I think the translation should be put back, or this line should be removed altogether